### ISSUE TITLE
simple gui & standalone install

### ISSUE THREAD

**Issue by patientx** [DakeQQ/F5-TTS-ONNX#11]
After being successful using my amd gpu with the help of you guys I decided to add a simple gui and standalone install (doesn't need FS-TTS and installs its own venv etc.)

https://github.com/patientx/F5-TTS-ONNX-gui

If you wish you can integrate the changes with your official app. ( Even without venv setup a gui makes it easier to use.) Also I linked to the latest onnx model's links you shared here, if that is a problem I can remove it. I can generate them myself and later upload to a few more places if you don't want your models shared.

**Comment by DakeQQ** [comment]
Yes, excellent work! Thank you for creating a GUI and standalone installer for the project. It's great to hear about your successful setup with your AMD GPU using the shared guidance.

Your initiative to enhance usability with a GUI is commendable, and we’re happy to recommend this excellent work. We’ll include a useful reference link to your repository so others can benefit from your contribution.

Thank you once again for your valuable efforts!

**Comment by dragitz** [comment]
Thank you for making the process way easier to use by automating it!

For future people, I ran into an issue that prevented me from using my own sample.wav, I just had to export it in mono instead of stereo (default option in audacity)

**Comment by DakeQQ** [comment]
Thank you for your kind hint! We truly appreciate your feedback and for sharing your solution with others. Your experience will surely help future users who may encounter the same issue.

To simplify the process, you can also apply the following code to convert a stereo audio file into mono:

```python
from pydub import AudioSegment
import soundfile as sf
import numpy as np

SAMPLE_RATE = 16000  # Default
stereo_audio = "./stereo_sample.wav"
mono_audio = "./mono_sample.wav"

audio = np.array(AudioSegment.from_file(stereo_audio).set_channels(1).set_frame_rate(SAMPLE_RATE).get_array_of_samples())
sf.write(mono_audio, audio, SAMPLE_RATE, format='WAVEX')
```
